### PR TITLE
Test case for the issue 280

### PR DESCRIPTION
**Pull request by meirka**

### CODE REVIEW

File: projects/testing-library/tests/issues/issue-280.spec.ts [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+  exports: [RouterModule]
+})
+class AppRoutingModule {}
+
+
+test('navigate to second page and back', async () => {
+  const subject = await render(MainComponent, {imports: [AppRoutingModule, RouterTestingModule]});
+  await subject.navigate('/');
+
+  expect(await screen.findByText('Navigate')).toBeTruthy();
```
Comment (meirka) [inline projects/testing-library/tests/issues/issue-280.spec.ts:44]: I left extra expects in place, showing that there is successful navigation to second page.
I typically don't do that in my tests, please let me know if you want me to remove extras.

Thank you.

File: projects/testing-library/tests/issues/issue-280.spec.ts [hunk 2]
```diff
@@ -0,0 +1,55 @@
+import {Component, NgModule} from '@angular/core';
+import {render, screen} from '../../src/public_api';
```
Comment (timdeschryver) [inline projects/testing-library/tests/issues/issue-280.spec.ts:2]:
```suggestion
import {render, screen} from '@testing-library/angular';
```

File: projects/testing-library/tests/issues/issue-280.spec.ts [hunk 3]
```diff
@@ -0,0 +1,55 @@
...
+import {RouterModule, Routes} from "@angular/router";
+import {RouterTestingModule} from "@angular/router/testing";
+import {click} from "@testing-library/user-event/dist/click";
```
Comment (timdeschryver) [inline projects/testing-library/tests/issues/issue-280.spec.ts:5]:
```suggestion
import userEvent from "@testing-library/user-event";
```

And then in the test it becomes:

```
userEvent.click(...);
```

File: projects/testing-library/tests/issues/issue-280.spec.ts (hunk 1 above)
Comment (timdeschryver) [reply projects/testing-library/tests/issues/issue-280.spec.ts:44]: This is fine, thanks!

**Review by timdeschryver — COMMENTED** [review]
Thanks for creating this PR!

I left some comments to make it consistent with the rest of the tests.
Could you have a look at them please? You can take your time for it, there's no rush.

File: projects/testing-library/tests/issues/issue-280.spec.ts (hunk 3 above)
Comment (meirka) [reply projects/testing-library/tests/issues/issue-280.spec.ts:5]: I don't see benefit making it explicit and longer, but will change.

File: projects/testing-library/tests/issues/issue-280.spec.ts (hunk 2 above)
Comment (meirka) [reply projects/testing-library/tests/issues/issue-280.spec.ts:2]: done

**Comment by meirka** [comment]
> left some comments to make it consistent with the rest of the test

No problem, made changes, let me know what you think.

Thanks!

**Review by timdeschryver — APPROVED** [review]
Great, thanks!

**Comment by timdeschryver** [comment]
@all-contributors please add @meirka  for bug, tests

**Comment by timdeschryver** [comment]
Oh yikes, it seems to be failing 😲

**Review by timdeschryver — COMMENTED** [review]
Sorry to bother again, but could you comment this test out and make a new one that succeeds by using the following technique.

```
  const router = TestBed.inject(Router);
  router.initialNavigation();
```

And a tip, instead of `toBeTruthy` you can use `toBeInTheDocument` which makes it easier to read imho.

**Comment by meirka** [comment]
> Sorry to bother again, but could you comment this test out and make a new one that succeeds by using the following technique.
> 
> ```
>   const router = TestBed.inject(Router);
>   router.initialNavigation();
> ```
> 
> And a tip, instead of `toBeTruthy` you can use `toBeInTheDocument` which makes it easier to read imho.

Sure, I can do that, but what's purpose? 
Is it just to pass the build?

**Comment by timdeschryver** [comment]
@meirka yea 😅

**Comment by meirka** [comment]
Sure, will do

**Comment by meirka** [comment]
Done, let know if anything else, don't hesitate.
PS: don't forget to include me into fix PR. Im curious.

**Review by timdeschryver — APPROVED** [review]
Thanks again 🤗

**Comment by meirka** [comment]
> Thanks again 🤗 I will ping you, and I hope to get this done next week.

Thank you for all the help, it was a pleasure!

**Pull request merged**